scripts/test-gui/index.mjs

```diff
@@ -122,6 +122,7 @@ function startApp (argv = []) {
   const forgeArgs = [ 'start', '--', `--data-dir="${CONF_DIRECTORY}"`, ...argv ]
   // Spawn's options: Use the root as CWD and pipe the process's stdio to the parent process.
   const spawnOptions = {
+    shell: process.platform === "win32",
```

Comment:
This could also be hardocoded to `true` rather than just for Windows.

Reply:
Adding to the comment I just made on the PR itself: I can see in the preview that we pass the `data-dir` argument. It can be that it works on UNIX this way because we use an entirely different spawn call, but I may be wrong. Anyways, as I outlined, I don't think there are any insurmountable security implications with just providing a hard-coded `true` here!